dw-ripple.js

```diff
@@ -68,32 +67,52 @@ export class DwRipple extends Ripple {
     this.__onTouchStart = this.__onTouchStart.bind(this);
     this.__onMouseUp = this.__onMouseUp.bind(this);
     this.__onTouchEnd = this.__onTouchEnd.bind(this);
-
-    parent.addEventListener('mouseenter', this._rippleHander.startHover);
-    parent.addEventListener('mouseleave', this._rippleHander.endHover);
-    parent.addEventListener('mousedown', this.__onMouseDown);
-    parent.addEventListener('touchstart', this.__onTouchStart);
-    parent.addEventListener('focus', this._rippleHander.startFocus);
-    parent.addEventListener('blur', this._rippleHander.endFocus);
+    this.__onPointerMove = this.__onPointerMove.bind(this);
+    this.__onTouchMove = this.__onTouchMove.bind(this);
+
+    parent.addEventListener("mouseenter", this._rippleHander.startHover);
+    parent.addEventListener("mouseleave", this._rippleHander.endHover);
+    parent.addEventListener("mousedown", this.__onMouseDown);
+    parent.addEventListener("touchstart", this.__onTouchStart);
+    parent.addEventListener("focus", this._rippleHander.startFocus);
+    parent.addEventListener("blur", this._rippleHander.endFocus);
+    window.addEventListener("pointermove", this.__onPointerMove);
+    window.addEventListener("touchmove", this.__onTouchMove);
   }
 
   __onMouseDown(e) {
-    window.addEventListener('mouseup', this.__onMouseUp);
-    this._rippleHander.startPress(e);
+    this.delayMouseDown = setTimeout(() => {
```

Comment:
Rename variable to `this._timeoutRippleOnMouseDown` and `_timeoutRippleOnTouchStart`